src/test/scala/org/opennetworkinsight/utilities/DomainProcessorTest.scala

```diff
@@ -1,68 +1,71 @@
 package org.opennetworkinsight.utilities
```

Comment:
crap

I dropped these tests in some merge noise

I will add them back and repush